.github/workflows/zowe-cli-plugin.yml

```diff
@@ -64,9 +60,7 @@ jobs:
     - name: Start K3s Cluster
       id: k3s
       if: ${{ always() && steps.build.outcome == 'success' && matrix.os == 'ubuntu-latest' }}
-      uses: debianmaster/actions-k3s@master
-      with:
-        version: latest
+      uses: nolar/setup-k3d-k3s@latest
```

Comment:
![image](https://github.com/zowe/zowe-cli-secrets-for-kubernetes/assets/112635587/70119079-7300-4a2d-921e-a032ecd40508)

Reply:
ideally we would want this to be locked down to a major version instead of accepting possible breaking changes with `latest` (e.g. when they go to v2)